lib/purchase.js

```diff
@@ -0,0 +1,26 @@
+const { Model, DataTypes } = require("sequelize");
+const sequelize = require("../config/connection");
+
+class Purchase extends Model {}
+
+Purchase.init(
+  {
+    price: {
+      type: DataTypes.INTEGER,
```

Comment:
type should be `DataTypes.DECIMAL(10,2)` since we're dealing with money and cents are involved : )